lib/cell.rb

```diff
@@ -1,10 +1,22 @@
 class Cell
   attr_reader :coordinate,
-              :ship
+              :ship,
+              :print_to_screen
```

Comment:
Do we need the `@print_to_screen` variable to be in the attr_reader? I don't think we will be calling it outside of the class.